dashboards-observability/public/components/application_analytics/home.tsx

```diff
@@ -0,0 +1,147 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ *
+ * Modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
+ */
+
+import React, { useEffect, useState } from 'react';
+import { AppTable } from './components/app_table';
+import { Application } from './components/application';
+import { CreateApp } from './components/create'
+import { Route, RouteComponentProps, Switch } from 'react-router';
+import { TraceAnalyticsComponentDeps, TraceAnalyticsCoreDeps } from '../trace_analytics/home';
+import { FilterType } from '../trace_analytics/components/common/filters/filters';
+import DSLService from 'public/services/requests/dsl';
+import PPLService from 'public/services/requests/ppl';
+import SavedObjects from 'public/services/saved_objects/event_analytics/saved_objects';
+import TimestampUtils from 'public/services/timestamp/timestamp';
+import { handleIndicesExistRequest } from '../trace_analytics/requests/request_handler';
+import { ObservabilitySideBar } from '../common/side_nav';
+
+interface HomeProps extends RouteComponentProps, TraceAnalyticsCoreDeps {
+  pplService: PPLService;
+  dslService: DSLService;
+  savedObjects: SavedObjects;
+  timestampUtils: TimestampUtils;
+}
+
+
+export type ApplicationType = {
+  name: string;
+  id: string;
+  composition: string;
+  currentAvailability: string;
+  availabilityMetrics: string;
+  dateCreated: string;
+  dateModified: string;
+};
+
+const dateString = new Date().toISOString();
+
+const dummyApplication: ApplicationType[] = [{
```

Comment:
I see a lot of references to `dummy` or `dummyApplication` of some sort, is that just hard-coded test data we are providing to App Analytics?

Reply:
It's just a placeholder I used for when I was just setting up the frontend. I'll replace it all once I'm done with the backend.